README.md

```diff
@@ -1,6 +1,196 @@
 # audit-reporting-maven-plugin
-Plugin to execute a series of audits, generate a result artifact, and report the results via the Maven site.
+Plugin to execute a series of one or more audits, generate a result artifact, and report the results via the Maven site.
 
 ## Examples
 
-Examples can be found underneath the integration test directory `src/it`
+Always up-to-date examples can be found underneath the integration test directory `src/it`.
+
+## Walkthrough
+
+To create a fresh set of audits, it is recommended to use more than one project. You can write any number of audit implementations; we recommend storing each implementation in separate projects but you may keep them together if desired. Finally, you will write a project to execute the audits and generate the result artifact and site.
+
+We'll start by writing our audit implementation. First, we'll need to make sure we have the following dependencies, at minimum:
+
+Auditor pom.xml
+```
+<dependency>
+  <groupId>org.infrastructurebuilder.audit</groupId>
+  <artifactId>auditor-api</artifactId>
+  <version>0.3.0</version>
+</dependency>
+<dependency>
+  <groupId>org.eclipse.sisu</groupId>
+  <artifactId>org.eclipse.sisu.inject</artifactId>
+  <version>0.3.4</version>
+</dependency>
```

Comment:
What do you need `sisu.inject` for?

Reply:
I don't know, but I respectfully request we kick this one down the road to my next PR set which will attempt to bring this up to date with the latest parent